docs/docs/spark-procedures.md

```diff
@@ -769,6 +769,10 @@ Here is a list of commonly used Spark read options:
 * `start-timestamp`: the exclusive start timestamp. If not provided, it reads from the table’s first snapshot inclusively.
 * `end-timestamp`: the inclusive end timestamp, default to table's current snapshot.                                                                                  
 
+!!! warning
+    `create_changelog_view` procedure rely on partitioning and sorting of all columns to identify carry-over rows when `identifer_columns` are not provided.
+    Keep in mind if table contains unorderable column (like Map or Binary type) and not configured with `identifer_columns`, this procedure will fail with AnalysisException.
```

Comment:
Just curious, binary type isn't a atomic type?https://github.com/apache/spark/blob/2ed6c3e511f322c5fd01953736c376a85ff2c687/sql/api/src/main/scala/org/apache/spark/sql/catalyst/expressions/OrderUtils.scala#L27

Reply:
sorry I just double checked and binary type can be ordered. I also moved to add precondition check instead of update doc here